ccd-definition/FixedLists/CMOActionStatus.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CMOActionStatus",
```

Comment:
Using status word in this context is bit wired to me. I'd say it is type of action that can be taken and since that I'd call it `CMOActionType`